### ISSUE TITLE
[Error]: Cannot stream audio from pulseaudio client:  "Audio-Jack-Status" missing

### ISSUE THREAD

**Issue by qtaped** [mikebrady/shairport-sync#1594]
### What happened?

Cannot stream audio from pulseaudio client, but it is working fine from iOS devices or macOS.

[edit]
**If I change the name of the server.**

I can see and connect to my shairport-sync server (running on last debian x86), but if I play music (with mocp or firefox for example) from a linux client (I tried with my laptop under archlinux and a raspberry under last debian), I am disconnected a few seconds then reconnected but the music is stuck.

I had an original Apple Airport, and it was working fine then. I already tried without any firewall (client and server sides). It was also working with an old fork called "shairplay".

Here is the journal error that pulseaudio return (same for both clients):

### (not so) Relevant log output

```shell
pulseaudio[1049]: "Audio-Jack-Status" missing in RTSP setup response
```

### Configuration Information.

Server:
Linux 4.19.0-16-amd64 #1 SMP Debian 4.19.181-1 (2021-03-19) x86_64 GNU/Linux
3.3.7-libdaemon-OpenSSL-Avahi-ALSA-jack-pa-dummy-stdout-pipe-soxr-convolution-metadata-mqtt-dbus-mpris-sysconfdir:/etc

or

 4.1.1-OpenSSL-Avahi-ALSA-soxr-sysconfdir:/etc

(a minimal debian with only Alsa, without pulse-audio nor jack)

Clients:
Linux 6.0.11-arch1-1 #1 SMP PREEMPT_DYNAMIC Fri, 02 Dec 2022 17:25:31 +0000 x86_64 GNU/Linux
pulseaudio 16.1

Linux 5.15.76-v8+ #1597 SMP PREEMPT Fri Nov 4 12:16:41 GMT 2022 aarch64 GNU/Linux
pulseaudio 14.2

### How did you install Shairport Sync?

A package manager (apt, apt install, yum, pkg, etc.)

### Check previous issues

- [X] Confirm

**Comment by mikebrady** [comment]
> I just realized that I do not use the latest release. I'll give a try.

Yeah, 3.3.7 is a bit ancient at this stage 😊.

**Comment by qtaped** [comment]
I installed 4.1 (without Airplay 2 support), and it worked, so I edited the conf file, and same bug appears. It seems that pulseaudio didnt like that the name of the server is not hostname.
`"Audio-Jack-Status" missing in RTSP setup response` still appears so I guess this was not relevant.
Another issue: the client does not close the connection when I switch to another audio output (my iPhone does).

But I can deal with that!

Thanks for the work you have done btw!
